examples/seismic/tutorials/09_snapshotting.ipynb

```diff
@@ -0,0 +1,806 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snapshotting with Devito using the `ConditionalDimension`\n",
+    "\n",
+    "This notebook intends to introduce new Devito users (especially with a C or FORTRAN background) to the best practice on saving snapshots to disk, as a binary float file. \n",
+    "\n",
+    "We start by presenting a naive approach, and then introduce a more efficient Devito-like method, explaining `ConditionalDimension`.\n",
+    "\n",
+    "This tutorial also saves a file named `snapshotting.mp4` in the current directory."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Initialize utilities"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%reset -f\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt \n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem Setup \n",
+    "The example presented in this tutorial is based on the codes from [the first TLE tutorial](https://github.com/opesci/devito/blob/master/examples/seismic/tutorials/01_modelling.ipynb) (Louboutin et. al., 2017), where one shot is modeled over a 2-layer velocity model. For more information about the example follow the link above.\n",
```

Comment:
Also, at the beginning:

"This tutorial is based on an example that has appeared in a [TLE tutorial], in which one shot is modeled over a 2-layer velocity model."